File: lib/easy/min_deletion_size.dart
int minDeletionSize(List<String> strs) {
  int count = 0;
  int rows = strs.length;
  int cols = strs[0].length;
  for (int col = 0; col < cols; col++) {
    int row = 1;
    while (row < rows) {
      if (strs[row - 1].codeUnitAt(col) > strs[row].codeUnitAt(col)) {
        count++;
        break;
      }
      row++;
    }
  }
  return count;
}

File: src/easy/min-deletion-size.ts
export const minDeletionSize = (strs: string[]): number => {
    let count = 0;
    let rows = strs.length;
    let cols = strs[0].length;
    for (let col = 0; col < cols; col++) {
        let row = 1;
        while (row < rows) {
            if (strs[row - 1][col] > strs[row][col]) {
                count++;
                break;
            }
            row++;
        }
    }
    return count;
};
